### PR TITLE
PAPP-23980: Enabled using "extract ioc" on password-encrypted PDF on FIPS instance.

### PR DESCRIPTION
**Pull request by paulnawat**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [x] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Enabled using "extract ioc" on password-encrypted PDF on FIPS instance.

## What is the current behavior? (OPTIONAL)
- Calling "extract ioc" on password-encrypted PDF will result in invalid cipher error. Only FIPS cipher is allowed here.

## What is the new behavior? (OPTIONAL)
- Enabled using "extract ioc" on password-encrypted PDF on FIPS instance.

---
Thanks for contributing!

### CODE REVIEW

**Comment by paulnawat** [comment]
Green build on OnPrem 5_1: /pipelines/4513565

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 beautifulsoup4==4.9.1
 defusedxml==0.7.1
-pdfminer.six==20211012
+git+https://github.com/phantomcyber/pdfminer.six.git@20211012-fips
```
Comment (paulnawat) [inline requirements.txt:3]: This is the key change. 
Here, I replaced all MD5 usage with SHA256.

File: parser.json [hunk 2]
```diff
@@ -478,5 +474,65 @@
             },
             "versions": "EQ(*)"
         }
-    ]
-}
\ No newline at end of file
+    ],
+    "pip39_dependencies": {
```
Comment (paulnawat) [inline parser.json:478]: These wheels are updated automatically by calling `pre-commit run --all-files`

File: requirements.txt (hunk 1 above)
Comment (paulnawat) [reply requirements.txt:3]: Reference: https://github.com/splunk-soar-connectors/ssh/pull/1/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R2

**Review by tonyc-phantom — APPROVED** [review]

**Review by brian-phantom — APPROVED** [review]

**Pull request merged**